### ISSUE TITLE
NPC Adventuring Fellow not being recalled in some zones after logging out/in

### ISSUE THREAD

**Issue by E5una** [EdenServer/community#1994]
### Checklist

- [ ] I have searched for existing issues for issues like this one. The issue has not been posted. (Duplicate reports slow down development.)
- [ ] I have provided reproducable steps. (No "as the title says" posts please. Provide reproducable steps even if it seems like a waste of time.)
- [ ] I have provided my client version in the details. (type /ver into your game window)

### Details

Logging in and out occasionally causes me to lose my Adventuring Fellow. They give the message as if they have been summoned for too long, even if it is only a few minutes. After talking to a dev it sounds like there is a flag, Zoning fellow style flag, that may not be set in all zones that allow an Adventuring Fellow. In the instance below where I took screenshots, expecting it to happen, I was in Yuhtunga Jungle. It has definitely happened to me before elsewhere though.

![Summoned](https://user-images.githubusercontent.com/13668758/63975168-5ad25500-ca7c-11e9-97e9-91bc25534cdc.PNG)
![Logging out](https://user-images.githubusercontent.com/13668758/63975170-5dcd4580-ca7c-11e9-9beb-2c978a81adab.PNG)
![Login](https://user-images.githubusercontent.com/13668758/63975172-5efe7280-ca7c-11e9-9bea-d926a779c2a5.PNG)

Version: 30181205_0

**Comment by E5una** [comment]
After the dev saw the screenshots they realized it’s probably not the flag, something to do with the timer instead. Just documenting conversation.

**Comment by E5una** [comment]
I have new additional information on this one that make it a bit more complicated. Two times before in Temple of Uggalepih, I've have issues where my NPC will no longer draw their weapon and attack when I do. To correct this in the past I zoned out into Yhoator Jungle, and then back in. 

Today I was leveling my NPC in temple when I had to zone because one of the Ninja tonberries (Tonberry Cutter) ran through a wall by the zone and then was "too far" to attack when it came out of the wall and stood on the stairs. I zoned, rested with my NPC, re-entered, but at this point my NPC stopped drawing its weapon. I decided to zone again, but this time my NPC immediately gave the time out messages shown previously in this ticket and then disappeared. 

![NPCSummon](https://user-images.githubusercontent.com/13668758/64175090-97a99d80-ce28-11e9-889d-6a37962a4d4b.PNG)
![NPCLeaving](https://user-images.githubusercontent.com/13668758/64175093-99736100-ce28-11e9-8eb2-91c7360b549a.PNG)
